### PR TITLE
Dept 40 shoutout icons

### PR DESCRIPTION
**Pull request by nkhorvash**
This card adds user icons to shoutout cards

A bug fix card will be added to the backlog for the small gap between the image and its border on some avatars.

A tech debt card will be added for testing as well.

A card will also be added to make the number of avatars displayed responsive to the width of the card, per conversation with Kathleen Gorman.

### CODE REVIEW

**Review by bdurel — CHANGES_REQUESTED** [review]
Hey Naysan, nice work. I have a few housekeeping notes - mostly just cleaning up the scss file. Also looking for some clarification. I believe that we decided to reverse the order of the icon stack so that it is in the order that you have it, and not the way it is on the figma (with the last avatar on top). Is that correct? I see a note suggesting it on Devlon's spike, but I don't see it mentioned elsewhere. Besides the scss notes, it looks good to me. As an aside, I know that you have a border that matches the figma, but I think it would look better with a thicker border (that actually looks like the figma)? Maybe 3 or 4 pixels? What do you think?

File: components/shoutouts-temp/shoutouts-comp.tsx [hunk 1]
```diff
@@ -79,6 +82,38 @@ const ShoutoutsComp = ({ shoutout }: IProps) => {
...
+          }}
+        >
+          {shoutout.recipients.map((recipient) => {
+            return (
+              <Avatar
+                className={styles.avatar}
+                key={recipient.employeeId}
+                src={recipient.image72}
+                sx={{
+                  '& .MuiAvatar-img': { width: '32px', height: '32px' },
```
[the remark this answers is not in the document]
Comment (nkhorvash) [reply components/shoutouts-temp/shoutouts-comp.tsx:103]: I can make it rems, but it's bigger than the container to try to fill the gap (which appeared to work partially, but not completely.

[the remark this answers is not in the document]
Comment (nkhorvash) [reply components/shoutouts-temp/shoutouts-comp.tsx:103]: Actually, that was true, but I changed it for something a little better. Not sure how it happened, but this is missing a few changes that should be in there. I'm fixing it now.

**Comment by nkhorvash** [comment]
There was a lot of cleanup to do, and a few bits of code that I thought were saved somehow were not in the project (like box-sizing). Now things look right to me, but please take another look.

I've also added a note to the card in JIRA to clarify the icon order, I was operating based on Devon's spike, but neglected to call out  that deviation from figma in my AC, good catch.

**Review by bdurel — APPROVED** [review]
Thanks for cleaning it up. Looks great and opperates as it should apart from the avatar bug. Has my approval. Well done! Also, if you're interested, I added the bug fix card to the backlog. My current suspicion is that borders aren't meant to be used with MUI's avatar, but who knows?

**Comment by Incarnate-Infinity** [comment]
@nkhorvash Largely looking good, just have a couple of thoughts.

Firstly, is 8 avatars and the "extra avatars" badge as a 9th the intended number of icons we want to have, or was the intention to have 10 total? Either way is fine, I just wanted to check and be sure. I can't recall if we had decided on an amount and 9 just feels odd to me somehow.

> Thanks for cleaning it up. Looks great and opperates as it should apart from the avatar bug. Has my approval. Well done! Also, if you're interested, I added the bug fix card to the backlog. My current suspicion is that borders aren't meant to be used with MUI's avatar, but who knows?

Secondly is in reference to @bdurel 's comment quoted above. I think the culprit may not be MUI's profile icon component but in fact the nextjs Image component. I only used the profile icon component a bit in the sandbox for the research spike on this but I don't believe I had the issue with the border gap, however I vividly recall dealing with that issue even with the custom-built icon component I made for DEPT1's version of the project, which did not use MUI's component but did use nextjs's Image component.

I could be wrong there, but I figured the information might be worth having. Once the first point is answered I think this'll be good to go!

**Comment by nkhorvash** [comment]
@Incarnate-Infinity I spoke with Kathleen about your question, and basically that is the max that will fit on a card at our smallest supported screen size. I will add a card today to make the visible icons responsive to screen size.

Also, excellent point about the bug! I added a comment to the bug card, Devon if you want to add to or amend that, please do.

**Review by Incarnate-Infinity — APPROVED** [review]
@nkhorvash That all makes sense, I just wanted to be sure on that!

LGTM 👍

**Comment by Incarnate-Infinity** [comment]
LGTM 👍

**Pull request merged**